frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetails.tsx

```diff
@@ -59,7 +59,7 @@ const PipelineDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath }) =
   }
   return (
     <>
-      <Drawer isExpanded={!!selectedId}>
+      <Drawer isExpanded={!!selectedId} className="odh-pipelines-task-drawer">
```

Comment:
You should import CSS files for classes you use. If you don't, cleanup can cause awkward removal of an import that just happens to work due to imports/inclusion of components

Reply:
This can be removed when the drawer layout is fixed.